README.md

```diff
@@ -1,4 +1,11 @@
-# Getting Started with Create React App
+# Math Magicias Project
+
+## Build with
+- Major languages: React Js Library
+- Technologies used: Linters, Git and Github, webpack and jest
+
+## Live Demo
+- Comming Soon...
```

Comment:
- [ ] Good job adding a professional README file as requested by the previous reviewer but the file is missing some required sections and contains extra content, Please make sure to add required sections and delete extra content from the file. I would suggest you use this [template](https://github.com/microverseinc/readme-template) but remember to customize it for your project.